### ISSUE TITLE
NSLScanner fails while creating symlinks

### ISSUE THREAD

**Issue by Sythog** [moraroy/NonSteamLaunchers-On-Steam-Deck#206]
https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/blob/f34088d4aec176eb9ce32c4d2d31b8ba2c57f9bd/NSLGameScanner.py#L466

I already have symlinks for the launchers I have installed named exactly as folder name in the folder_names dictionary. The script enters the if condition on L466 and attempts to rename the symlink to the appid on L475. Of course, it fails and no shortcuts are created :(

**Comment by Sythog** [comment]
@moraroy hey, do you have any thoughts and/or plans on that? 
please don't hesitate to let me know if you need any help. I'm not a python guru, but I know a thing or two.
I just need a general direction on where to start with this if you'd like me to make a contribution

**Comment by moraroy** [comment]
@Sythog yes I do actually, currently concentrating in making the decky plugin version a but better sorry I haven't got back to you. I do have a plan actually I would like to get rid of the symlinks in the compatdata folder all together. You have discord? Message me @moraroy

**Comment by Sythog** [comment]
Is being addressed in #210
